src/main/kotlin/app/extractors/RustExtractor.kt

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Sourcerer Inc. All Rights Reserved.
+// Author: Alexander Surkov (alex@sourcerer.io)
+
+package app.extractors
+
+import app.model.CommitStats
+import app.model.DiffFile
+
+class RustExtractor : ExtractorInterface {
+    companion object {
+        const val LANGUAGE_NAME = Lang.Rust
+        val evaluator by lazy {
+            ExtractorInterface.getLibraryClassifier(LANGUAGE_NAME)
+        }
+        val importRegex = Regex("^extern crate \\w+;$")
+        val commentRegex = Regex("(////.+$)|(//[*].*?[*]//)")
```

Comment:
Are there four slashes in rust?!

Reply:
two as everywhere, just every backslash is backslashed, """ will help I think

Reply:
backslashed, not forwardslashed maybe?

Reply:
right, for some reason I "backslashed" them too :) we should have a test on this.

Reply:
wat :)